### PR TITLE
BLorient829*

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/974
I am in favour of deleting BLorient829a and BLorient829b and keeping only this summary record for the current unit of circulation.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 1]
```diff
@@ -49,7 +77,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     <listBibl type="catalogue">
                                         <bibl>
                                             <ptr target="bm:Wright1877BM"/>
-                                            <citedRange unit="page">22a; 334b</citedRange>
+                                            <citedRange unit="page">334b</citedRange>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:80]: should this not repeat all pages for the parts? just wondering

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 2]
```diff
@@ -60,7 +88,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <msIdentifier>
                             <repository ref="INS00001BL"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:89]: it is better to not repeat the repository for each part

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 3]
```diff
@@ -60,7 +88,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                             <collection>Oriental</collection>
-                            <idno>BL Oriental 829, ff. 1-2</idno>
+                            <idno>BL Oriental 829*, ff. 1-2</idno>
                             <altIdentifier>
                                 <idno>Wright cat. CCCVI</idno>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:93]: 306 or 406?

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 4]
```diff
@@ -124,7 +153,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <msIdentifier>
                             <repository ref="INS00001BL"/>
                             <collection>Oriental</collection>
-                            <idno>BL Oriental 829, f. 3</idno>
+                            <idno>BL Oriental 829*, f. 3</idno>
                             <altIdentifier>
                                 <idno>Wright cat. CCCVII</idno>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:158]: 307 or 407?

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 5]
```diff
@@ -186,7 +216,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <msIdentifier>
                             <repository ref="INS00001BL"/>
                             <collection>Oriental</collection>
-                            <idno>BL Oriental 829, f. 4</idno>
+                            <idno>BL Oriental 829*, f. 4</idno>
                             <altIdentifier>
                                 <idno>Wright cat. CCCVIII</idno>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:221]: 308 or 408?

File: LondonBritishLibrary/orient/BLorient829fragments.xml [hunk 6]
```diff
@@ -589,11 +622,388 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        </additional>
+                    </msPart>
+                    
+                    <msPart xml:id="p10">
+                        <msIdentifier>
+                            <repository ref="INS00001BL"/>
+                            <collection>Oriental</collection>
+                            <idno>BL Oriental 829, ff. 26-32</idno>
+                            <altIdentifier>
+                                <idno>Wright cat. </idno>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient829fragments.xml:745]: no number?

**Review by eu-genia — CHANGES_REQUESTED** [review]
wow! 
agree to leaving only the "folder" and removing the other records

File: LondonBritishLibrary/orient/BLorient829fragments.xml (hunk 6 above)
Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient829fragments.xml]: No, a few parts are only listed in the index and don't have a description...

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**